### ISSUE TITLE
Deployment onto production

### ISSUE THREAD

**Issue by solfisher** [Sosasees/tenacity-webpage#16]
Seeing as this repo hasn't been updated for a while, I've taken the task of cleaning up all of the HTML/CSS and making a production-ready draft. You can view it here:

https://cronut.cafe/~sfr/tenacity

and its source code is hosted on sourcehut:

https://git.sr.ht/~sfr/tenacityaudio.org

A few notes:
- The buttons are wider than before, and they're sorted as one row. This still looks good imo, and cuts a whole lot of unnecessary CSS.
- I've cleaned up most of the classes and variables.
- I've rearranged the buttons to fit in with the paragraphs.
- I've condensed everything into one commit, since this history tree is too messy for me to bother with fixing.

@Sosasees: if this looks fine, I'm going to push this into production.

**Comment by solfisher** [comment]
cc @notaproton @panos @semisol @akleja, please give your feedback

**Comment by panos** [comment]
- Key should be more lowkey.
- Website content should preferably be licensed under a Creative Commons license.

**Comment by solfisher** [comment]
> * Website content should preferably be licensed under a Creative Commons license.

There's not much content to license under CC.

> * Dark mode with an alternating screenshot, maybe?

It should use the correct screenshot depending on theme.

**Comment by NotAProton** [comment]
https://github.com/Sosasees/tenacity-webpage/blob/96741428ef279b999fbc3a2c58011ebae72e0466/index.html#L156

Prepackaged -> Pre-packaged

also, what about internationalisation?

**Comment by solfisher** [comment]
> also, what about internationalisation?

i18n can be dealt with later, preferably by whoever is willing to translate the website to their language.

> Screenshot needs update.

The screenshot is linked to the tenacityteam/assets repository, so updating it there should fix it.

> Key should be more lowkey.

I made the `<pre>` tag smaller -- how does it look [now](https://cronut.cafe/~sfr/tenacity)?

**Comment by Sosasees** [comment]
Looks very great.
This website is ready to be published.

**Comment by akleja** [comment]
Looks good! Looks nice in elinks as well.
The only thing I'm a bit unsure about is the inclusion of Kay, who I thought of more as an easter-egg. Especially in a text based browser it becomes very prominent.

**Comment by solfisher** [comment]
I think it's fine being there, at least on the HTML version. It's not in your face, and you need to scroll all the way to the end to see it. Not hidden by any means, sure, but also not plastered in the center of the page.

**Comment by Sosasees** [comment]
I just noticed that [the page with the old design has been subtly updated](https://web.archive.org/web/20211001203525/https://tenacityaudio.org/) to include a link to [Mastodon](https://fosstodon.org/@tenacity).

Right now is very likely not a good time to make changes to the new page since it's in the middle of deployment,
so all I can do is
remind you of this missing link.

**Comment by akleja** [comment]
@solfisher Fair enough.
Just one more nitpick. I think it looks a bit strange to repeat the gradient of the icon on the text of the logo. I think it would look better with a solid color on the text. But it's not that big of a deal, just thought I'd share my thoughts on it.

**Comment by solfisher** [comment]
Thanks for the feedback. Everything looks relatively minor, so I'll push
this onto the website as-is and tend to it later.

**Comment by Semisol** [comment]
![image](https://user-images.githubusercontent.com/45574030/135703978-f7aa1836-43d3-48fe-aa04-bd7ef8d0c183.png)
Please add chickens to the page thank you
Otherwise, just make Key less apparent but still visible, like gray it out a little bit I guess?

**Comment by Sosasees** [comment]
> ![image](https://user-images.githubusercontent.com/45574030/135703978-f7aa1836-43d3-48fe-aa04-bd7ef8d0c183.png) Please add chickens to the page thank you

I smell "design by committee" here.
every addition must have a good reason.

the website has "Key the cat" hidden at the bottom
because they're the Tenacity mascot. "Chickens" have nothing to do with Tenacity.

**Comment by solfisher** [comment]
> I just noticed that [the page with the old design has been subtly updated](https://web.archive.org/web/20211001203525/https://tenacityaudio.org/) to include a link to [Mastodon](https://fosstodon.org/@tenacity).

Yes, Mastodon has support for "verified" URLs, where if you link to it, it shows a checkmark.
I've sent a patch which adds Mastodon and Twitter links to the footer.
